restapi/src/main/java/io/stargate/web/docsapi/dao/DocumentDB.java

```diff
@@ -145,20 +143,8 @@ public boolean treatBooleansAsNumeric() {
     return !dataStore.supportsSecondaryIndex();
   }
 
-  public static List<String> getForbiddenCharactersMessage() {
-    return forbiddenCharacters.stream().map(ch -> "`" + ch + "`").collect(Collectors.toList());
-  }
-
-  public static boolean containsIllegalChars(String x) {
-    return forbiddenCharacters.stream().anyMatch(ch -> x.indexOf(ch) >= 0);
-  }
-
-  public static String replaceIllegalChars(String x) {
-    String newStr = x;
-    for (Character y : forbiddenCharacters) {
-      newStr = newStr.replace(y, '_');
-    }
-    return newStr;
+  public static boolean containsIllegalSequences(String x) {
+    return x.contains("[") || UNICODE_ESCAPE_PATTERN.matcher(x).find();
```

Comment:
Might not matter a lot but these 2 checks can be combined into single regexp. Or, since UNICODE_ESCAPE_PATTERN looks like constant String (unless I misread it -- so many backslashes :) ), could just use `contains()`. But java.util Regexps have more overhead than simple substring check.

(actually, funnily enough I was playing with this part of code & test performance for fun:

https://github.com/tatu-at-datastax/stargate-micro-benchmarks

... but I digress)

Reply:
Oh yea, I was doing something more complex before, but now it can just be a simple `contains` :) I will change